### ISSUE TITLE
Agreeing to the Contributor Licence Agreement

### ISSUE THREAD

**Issue by Coda-Coda** [Coda-Coda/Veracity-Logic-Mechanised#1]
If you wish to contribute to this repository, please review the [Contributor Licence Agreement](https://github.com/Coda-Coda/Veracity-Logic-Mechanised/blob/main/CLA.md) named [CLA.md](https://github.com/Coda-Coda/Veracity-Logic-Mechanised/blob/main/CLA.md) in the root of the repository and indicate your agreement below, such as saying "I agree to the CLA v1.0". Please indicate the version of the CLA you are agreeing to by its version number and/or git commit hash.

This is a prerequisite to being added as a contributor to the repository or having your pull requests merged.

**Comment by Coda-Coda** [comment]
I agree to the [CLA v1.0](https://github.com/Coda-Coda/Veracity-Logic-Mechanised/blob/5a2d3a692fc3b1e79f79a9268dc9775084d246e3/CLA.md) (this also applies to my past commits to this repository).

**Comment by stever00** [comment]
I agree to the [CLA v1.0](https://github.com/Coda-Coda/Veracity-Logic-Mechanised/blob/5a2d3a692fc3b1e79f79a9268dc9775084d246e3/CLA.md) (this also applies to my past commits to this repository).

**Comment by camBrown99** [comment]
I agree to the [CLA v1.0](https://github.com/Coda-Coda/Veracity-Logic-Mechanised/blob/5a2d3a692fc3b1e79f79a9268dc9775084d246e3/CLA.md) (this also applies to my past commits to this repository).
